paparazzi/api/paparazzi.api

```diff
@@ -0,0 +1,255 @@
+	public final fun component2 ()Ljava/lang/String;
+	public final fun component3 ()Ljava/lang/String;
+	public final fun component4 ()I
+	public final fun component5 ()Ljava/util/List;
+	public final fun component6 ()Ljava/util/List;
+	public final fun component7 ()Ljava/util/List;
+	public final fun component8 ()Ljava/util/List;
+	public final fun component9 ()Ljava/util/List;
+	public final fun copy (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;ILjava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;)Lapp/cash/paparazzi/Environment;
+	public static synthetic fun copy$default (Lapp/cash/paparazzi/Environment;Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;ILjava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;ILjava/lang/Object;)Lapp/cash/paparazzi/Environment;
+	public fun equals (Ljava/lang/Object;)Z
+	public final fun getAllModuleAssetDirs ()Ljava/util/List;
+	public final fun getAppTestDir ()Ljava/lang/String;
+	public final fun getCompileSdkVersion ()I
+	public final fun getLibraryAssetDirs ()Ljava/util/List;
+	public final fun getLibraryResourceDirs ()Ljava/util/List;
+	public final fun getLocalResourceDirs ()Ljava/util/List;
+	public final fun getModuleResourceDirs ()Ljava/util/List;
+	public final fun getPackageName ()Ljava/lang/String;
+	public final fun getPlatformDir ()Ljava/lang/String;
+	public final fun getResourcePackageNames ()Ljava/util/List;
+	public fun hashCode ()I
+	public fun toString ()Ljava/lang/String;
+}
+
+public final class app/cash/paparazzi/EnvironmentKt {
+	public static final fun androidHome ()Ljava/lang/String;
+	public static final fun detectEnvironment ()Lapp/cash/paparazzi/Environment;
+}
+
+public final class app/cash/paparazzi/Flags {
+	public static final field $stable I
+	public static final field DEBUG_LINKED_OBJECTS Ljava/lang/String;
+	public static final field INSTANCE Lapp/cash/paparazzi/Flags;
+}
+
+public final class app/cash/paparazzi/HtmlReportWriter : app/cash/paparazzi/SnapshotHandler {
+	public static final field $stable I
+	public fun <init> ()V
+	public fun <init> (Ljava/lang/String;)V
+	public fun <init> (Ljava/lang/String;Ljava/io/File;)V
+	public fun <init> (Ljava/lang/String;Ljava/io/File;Ljava/io/File;)V
+	public synthetic fun <init> (Ljava/lang/String;Ljava/io/File;Ljava/io/File;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun close ()V
+	public fun newFrameHandler (Lapp/cash/paparazzi/Snapshot;II)Lapp/cash/paparazzi/SnapshotHandler$FrameHandler;
+}
+
+public final class app/cash/paparazzi/InstantAnimationsRule : org/junit/rules/TestRule {
+	public static final field $stable I
+	public fun <init> ()V
+	public fun apply (Lorg/junit/runners/model/Statement;Lorg/junit/runner/Description;)Lorg/junit/runners/model/Statement;
+}
+
+public final class app/cash/paparazzi/Paparazzi : org/junit/rules/TestRule {
+	public static final field $stable I
```

Comment:
who dis?

Reply:
good question.  i don't know.  @JakeWharton @drewhamilton ?

Reply:
The Compose compiler added it. Not sure why it isn't marked synthetic.